akka-docs-new/src/main/paradox/guide/quickstart.md

```diff
@@ -0,0 +1,113 @@
+# Your First Akka Application - Hello World
+
+After all this introduction, we are ready to build our first actor system. We will do this in three chapters. 
+In this first chapter we will help you to set up your project and tools and have a simple "Hello World" demo running. 
+We will keep this to the bare minimum and then extend the sample application in the next chapter. Finally we review 
+what we have learned in the third chapter, looking in detail how the pieces work and fit together.
+
+> Our goal in this chapter is to set up a working environment for you, create an application that starts up and stops 
+an ActorSystem and create an actor which we will test.
+
+As the very first thing, we need to make sure that we can compile our project and have a working IDE setup to be 
+able to edit code comfortably. Depending on your preference for build tool and IDE there are multiple paths you can 
+follow here.
+ 
+## Setting up the build
+
+Depending on your build tool, you need to set up the layout for your project and tell the build tool about your 
+dependencies (libraries that you want to use). There are common things to care for independently of your choice 
+of build tool:
+
+* Declare `akka-actor` as a dependency. This is the core library of Akka, the one we are now learning
+* Declare `akka-testkit` as a dependency. This is a toolkit for testing Akka applications. Without this 
+  dependency you will have a hard time testing actors.
+* **Use the latest Akka version for new projects (unless you have additional constraints)!**  
+* **Don’t mix Akka versions! You are free to use any Akka version in your project, but you must use 
+    that version for all Akka core projects** In this sample it means that `akka-actor` and `akka-testkit` should 
+    always be of the same version.
+
+### sbt
+
+If you plan to use sbt, your first step is to set up the directory structure for the project. sbt follows the 
+directory layout standard of Maven. Usually, you want to start with the following directories and files:
+
+* `/src`
+  * `/main` (this is where main, production classes live)
+    * `/scala` (this is where Scala classes live)
+    * `/java` (this is where Java classes live if you plan to use Java as well)
+    * `/resources` (this is where non-source files live which are required on the classpath. 
+    Typical example is application.conf which contains the configuration for your application. 
+    We will cover this in detail in CONFIG-SECTION)
+  * `/test`
+    * `/scala` (this is where Scala test and test helper classes live)
+    * `/java` (this is where Java test and test helper classes live if you plan to use Java as well)
+    * `/resources` (this is where non-source files live which are required on the classpath to run tests. 
+    Typically this contains an application.conf that overrides the default configuration for tests. We will 
+    cover this in detail in CONFIG-SECTION)
+  * `project`
+    * `build.properties` ()
```

Comment:
migrating this to SBT-new g8, I think is better in the end. So let's keep this as it is for now and rewrite it to the new stuff.